Lib/statistics.py

```diff
@@ -214,17 +214,15 @@ def _ss(data, c=None):
         T, total, count = _sum((d := x - c) * d for x in data)
         return (T, total, count)
     count = 0
-    sx_partials = {}
-    sx_partials_get = sx_partials.get
-    sxx_partials = {}
-    sxx_partials_get = sxx_partials.get
+    sx_partials = defaultdict(int)
+    sxx_partials = defaultdict(int)
```

Comment:
Cool! This saves me the effort of posting code showing that `defaultdict` actually runs faster on my box under 3.10.1, even when all keys are missing :wink:.